### ISSUE TITLE
Add "home" tab

### ISSUE THREAD

**Issue by YaManicKill** [thelounge/lounge#791]
As discussed in #503, we could make a "home" tab which will initially be blank, so that when we close a channel that we are currently on, we would go back to the "home" tab so we don't mark any other channel as read. This is how slack does it, fwiw.

This also has a future advantage of having some other possibilities for viewing things (highlights, stats, etc) further down the line, but initially it should probably just be blank.

**Comment by MaxLeiter** [comment]
Similar to #650

**Comment by YaManicKill** [comment]
Yeah, it's not exactly the same, but that could definitely tie in. I assume
that they are influenced by slack having one where its just basically a pm
to yourself, but the idea is that it wouldn't go over irc, I guess?

**Comment by teamcoltra** [comment]
This could be a server tab, which most IRC clients already have.

**Comment by YaManicKill** [comment]
@teamcoltra We already have a server tab, which is slightly different from this. The server tab is for server notices, motd, etc.
